American Whitewater/Helpers/AWApiReachHelper.swift

```diff
@@ -0,0 +1,620 @@
+import UIKit
+import CoreData
+import Alamofire
+import SwiftyJSON
+import Foundation
+import CoreLocation
+
+/*
+ This is the new version of the API Helper Singleton
```

Comment:
This comment can be updated to reflect what's here now.

Reply:
Updated.